client/src/components/v2/charts/TextModeChart.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Table } from 'react-bootstrap';
+import { useData } from 'api/v2/sensors';
+import { usePowerModel } from 'api/v2/powerModel';
+import styles from './TextModeChart.module.css';
+
+/**
+ * Text Mode chart component
+ *
+ * @returns {React.Component} Component
+ */
+export default function TextModeChart() {
+  const data = useData();
+  const powerModelData = usePowerModel();
+
+  /**
+   * Format a value into text
+   *
+   * @param {?number} value Value to format. May be null
+   * @param {string}  unit  Unit of measurement
+   * @returns {string} Formatted value
+   */
+  function formatValue(value, unit) {
```

Comment:
We definitely need unit tests for all of these components. See seperate card for this.